### PR TITLE
Refine WebGLRenderingContext.p.bindFramebuffer doc

### LINKED ISSUE #2435: Issue with "WebGLRenderingContext.bindFramebuffer()": No description of behavior when framebuffer is null
MDN URL: https://developer.mozilla.org/en-US/docs/Web/API/WebGLRenderingContext/bindFramebuffer

#### What information was incorrect, unhelpful, or incomplete?

Does not describe that `framebuffer` is optional, and that passing null returns the target to being the canvas.

#### Specific section or headline?

Parameters, primarily (though an example may be of use)

#### What did you expect to see?

Wording along the lines of "If `framebuffer` is `null`, then the canvas (which has no `WebGLFramebuffer` object) is bound."

#### Did you test this? If so, how?

I have not particularly tested this right now.
However, confirmation of this can be found in the WebGL specification - where the canvas is described as "default framebuffer provided by the context" - as linked by the page.

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/web/api/webglrenderingcontext/bindframebuffer`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/API/WebGLRenderingContext/bindFramebuffer
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/api/webglrenderingcontext/bindframebuffer/index.html
* Last commit: https://github.com/mdn/content/commit/2d423bd1a98b85b8584f08990792ebf39fcea56b
* Document last modified: 2021-01-09T13:41:23.000Z

</details>

### PR DESCRIPTION
**Pull request by CreaTorAlexander**
Fixes #2435

### CODE REVIEW

File: files/en-us/web/api/webglrenderingcontext/bindframebuffer/index.html [hunk 1]
```diff
@@ -40,7 +40,7 @@ <h3 id="Parameters">Parameters</h3>
     </ul>
   </dd>
   <dt>framebuffer</dt>
-  <dd>A {{domxref("WebGLFramebuffer")}} object to bind.</dd>
+  <dd>If <code>framebuffer</code> is null, then the canvas (which has no {{domxref("WebGLFramebuffer")}} object) is bound.</dd>
```
Comment (sideshowbarker) [inline files/en-us/web/api/webglrenderingcontext/bindframebuffer/index.html:43]: What about if `framebuffer` isn’t null?

Comment (CreaTorAlexander) [reply files/en-us/web/api/webglrenderingcontext/bindframebuffer/index.html:43]: I am not familiar with framebuffers but isn't the rest of the site about about framebuffer when its not null?

Comment (sideshowbarker) [reply files/en-us/web/api/webglrenderingcontext/bindframebuffer/index.html:43]: > I am not familiar with framebuffers but isn't the rest of the site about about framebuffer when its not null?

The text here needs to describe both cases, at point of use.

@20kdc can you help here please?

Comment (20kdc) [reply files/en-us/web/api/webglrenderingcontext/bindframebuffer/index.html:43]: It arguably should have both.
```
  <dd>
    A {{domxref("WebGLFramebuffer")}} object to bind.
    If null, then the canvas (which has no {{domxref("WebGLFramebuffer")}} object) is bound.
  </dd>
```

Comment (CreaTorAlexander) [reply files/en-us/web/api/webglrenderingcontext/bindframebuffer/index.html:43]: Ah okay :+1:  I will update it later on

**Review by sideshowbarker — APPROVED** [review]

**Pull request merged**